scripts/sidebarJsonToYaml.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const yaml = require('js-yaml')
+const slugify = require('slugify')
+const sidebarJSON = require('../content/_data/sidebar.json')
+const sidebarKeys = Object.keys(sidebarJSON)
+let formattedSidebar = {}
+
+const process = (arr) => {
+  return arr.reduce((acc, { slug, children, title }) => {
+    return {
+      ...acc,
+      [slug ? slug : slugify(title).toLowerCase()]: children
+        ? process(children)
+        : `${slug ? slug : slugify(title).toLowerCase()}`,
+    }
+  }, {})
+}
+
+sidebarKeys.map((key) => {
```

Comment:
Check this out: instead of manually iterating over all the top-level keys of `sidebarJSON` and then processing the corresponding array value, which you then manually merge into the `formattedSidebar` object, why not just get an array of all those array values with [`Object.values()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) and then flatten that array using [`Array.prorotype.flat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) into something that `process()` understands?

```js
const formattedSidebar = process(Object.values(sidebarJSON).flat())
```